patches/chrome-browser-profiles-profile_manager.h.patch

```diff
@@ -1,12 +1,12 @@
 diff --git a/chrome/browser/profiles/profile_manager.h b/chrome/browser/profiles/profile_manager.h
-index 8ba4e0ba9f3c014fb94794d50afe8d54e94e3fdf..9e0fd8ccd3e5d2f5bf78c3b66ff5ef89f318d5a7 100644
+index 8ba4e0ba9f3c014fb94794d50afe8d54e94e3fdf..4551ca035aabe5844d3676bb093678ada0099c7f 100644
 --- a/chrome/browser/profiles/profile_manager.h
 +++ b/chrome/browser/profiles/profile_manager.h
-@@ -349,6 +349,7 @@ class ProfileManager : public Profile::Delegate {
-   void set_do_final_services_init(bool do_final_services_init) {
-     do_final_services_init_ = do_final_services_init;
-   }
-+  BRAVE_PROFILE_MANAGER_H
+@@ -355,6 +355,7 @@ class ProfileManager : public Profile::Delegate {
+       std::unique_ptr<AccountProfileMapper> mapper);
+ #endif
  
- #if BUILDFLAG(IS_CHROMEOS_LACROS)
-   void SetAccountProfileMapperForTests(
++  BRAVE_PROFILE_MANAGER_H
+  private:
+   friend class TestingProfileManager;
```

Comment:
`#define TestingProfileManager TestingProfileManager; friend class BraveProfileManager;`

Reply:
Fixed in bc012a12237f8ce3735467e076624e77b00cca3f